R/readQFeatures.R

```diff
@@ -25,12 +32,11 @@
 ##' `runCol` parameters, whether passed by the `colData` `data.frame`
 ##' and/or the `quantCols` and `runCol` arguments.
 ##'
-##' ## Single-set case
+##' @section Single-set case:
```

Comment:
Not sure we need a `@section` tag here, as we want this to be a sub-section within the Details section, not a new one.

Reply:
my bad, reverting to title